galley/cmd/gals/cmd/validator.go

```diff
@@ -56,7 +56,7 @@ func createMixerValidator(kubeconfig string) (store.BackendValidator, error) {
 	if err != nil {
 		return nil, err
 	}
-	rv, err := validator.New(checker.NewTypeChecker(), "", s, adapters, templates)
+	rv, err := validator.NewValidator(checker.NewTypeChecker(), "", s, adapters, templates)
```

Comment:
stutter? why not `New` like it was before?

Reply:
Will fix